packages/traveling-fastlane/actions/validate_apple_credentials.rb

```diff
@@ -9,7 +9,24 @@
 json_reply = with_captured_stderr{
   begin
     account = Spaceship::Portal.login($appleId, $password)
-    $stderr.puts (JSON.generate({result:'success', teams:account.teams}))
+
+    itc_cookie_content = Spaceship::Portal.client.store_cookie
+    cookies = YAML.safe_load(
+      itc_cookie_content,
+      [HTTP::Cookie, Time], # classes whitelist
+      [],                   # symbols whitelist
+      true                  # allow YAML aliases
+    )
+    cookies.select! do |cookie|
+      cookie.name.start_with?("myacinfo") || cookie.name == 'dqsid'
```

Comment:
out of curiosity: what's `dqsid`?

Reply:
https://github.com/expo/expo-cli/pull/249/files#diff-a810cfaca51ba1061d279298a877bfcbR13
https://github.com/expo/expo-cli/pull/249/files#diff-a810cfaca51ba1061d279298a877bfcbR25